build.gradle

```diff
@@ -193,67 +193,6 @@ test {
     systemProperty 'junit.jupiter.extensions.autodetection.enabled', 'true'
 }
 
-tasks.register("writeBranchName") {
-    // Define an output stream to write to instead of terminal
-    def stdout = new ByteArrayOutputStream()
-
-    // Execute the git command
-    exec {
-        commandLine "git", "rev-parse", "--abbrev-ref", "HEAD"
-        // Write to the output stream instead of terminal
-        standardOutput = stdout
-    }
-
-    // Parse the output into a string
-    def outstream = stdout.toString().trim()
-
-    // Create a new file
-    new File(
-            // Join project directory and deploy directory
-            projectDir.toString() + "/src/main/deploy",
-            // File name to write to
-            "branch.txt"
-            ).text = outstream // Set the contents of the file to the variable branch
-}
-
-deploy.targets.roborio.artifacts.frcStaticFileDeploy.dependsOn(writeBranchName)
-
-tasks.register("writeCommitHash") {
-    def stdout = new ByteArrayOutputStream()
-
-    exec {
-        commandLine "git", "rev-parse", "--short", "HEAD"
-        standardOutput = stdout
-    }
-
-    def outstream = stdout.toString().trim()
-
-    new File(
-            projectDir.toString() + "/src/main/deploy",
-            "commit.txt"
-            ).text = outstream
-}
-
-deploy.targets.roborio.artifacts.frcStaticFileDeploy.dependsOn(writeCommitHash)
-
-tasks.register("writeComputerName") {
-    def stdout = new ByteArrayOutputStream()
-
-    exec {
-        commandLine "hostname"
-        standardOutput = stdout
-    }
-
-    def outstream = stdout.toString().trim()
-
-    new File(
-            projectDir.toString() + "/src/main/deploy",
-            "hostname.txt"
-            ).text = outstream
-}
-
-deploy.targets.roborio.artifacts.frcStaticFileDeploy.dependsOn(writeComputerName)
-
 // tasks.register("writeBuildDate") {
 //     def stdout = new ByteArrayOutputStream()
```

Comment:
Why is this commented out?  Assume it was commented out in the version, but let's go ahead and remove it

Reply:
removed, resolving